### PR TITLE
Drop python 3.8

### PR DESCRIPTION
**Pull request by AlyssaCote**
Python 3.8 is no longer going to be supported because it's near end of life. I've gone through and removed/updated everything that I could find! I added `no-merge` because I am not sure if there's a specific order that we need to merge all the repos.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -31,7 +31,7 @@ build-backend = "setuptools.build_meta"
 
 [tool.black]
 line-length = 88
-target-version = ['py38', 'py39', 'py310']
+target-version = ['py39', 'py310']
```
Comment (MattToast) [inline pyproject.toml:34]: Not strictly related to this PR, but can we add `'py311'` to this list while we are here?

**Review by MattToast — APPROVED** [review]
One small request, but nothing worth holding up approval over! LGTM!!

File: pyproject.toml (hunk 1 above)
Comment (AlyssaCote) [reply pyproject.toml:34]: you got it!

**Review by mellis13 — APPROVED** [review]
Thank you!  LGTM

**Pull request merged**